.github/actions/test_tests-cache.sh

```diff
@@ -3,7 +3,16 @@ set -e -u -x -o pipefail
 
 TMP_CACHE_DIR=$(mktemp -d -t glpi-cache-test-XXXXXXXXXX)
 
-for CONFIG in {"--use-default","--dsn=memcached://memcached","--dsn=redis://redis"}; do
+PHP_MAJOR_VERSION="$(echo $PHP_VERSION | cut -d '.' -f 1,2)"
+if [[ "$PHP_MAJOR_VERSION" -ne "8.3" ]]; then
+  CONFIGS=( "--use-default" "--dsn=memcached://memcached" "--dsn=redis://redis" )
+else
+  # test with redis adapter does not pass on PHP 8.3 alpha 1
+  # do not know if it is due to a PHP bug, a redis extension bug or a symfony bug
```

Comment:
Maybe fixed by https://github.com/php/php-src/commit/49fbbea2ea5fe1f6bab0719c95a46a119ea3d91f

Waiting for next prerelease to be able to confirm.

Reply:
Redis test suite is OK with PHP 8.3-alpha2.